README.md

```diff
@@ -51,31 +71,57 @@ This will generate the following JSON output:
+Location: <a href="/.cloud/.azure/workspace.json">`/.cloud/.azure/workspace.json"` file</a> in your repository
 
-Once you save your changes to the file, the predefined GitHub workflow that trains and deploys a model on Azure Machine Learning gets triggered. Check the actions tab to view if your actions have successfully run.
+This is necessary so that your GitHub Actions will connect to your desired AzureML workspace. Click on the link and make your edits:
+
+```sh
+{
+   #same value for your-workspace-name as above
+   "name": "your-workspace-name",
+   #same resource group as above
+    "resource_group": "your-resource-group",
+   #this stays the same
+    "create_workspace": true
+}
+```
+If you want the Action to create a new workspace in that resource group, 
+pick a name for your new workspace, and assign it to the `name` parameter. 
+You can also delete the `name` parameter, if you want the action to use the defau
+value, which is the repository name.
+
+Once you save your changes to the file, the predefined GitHub workflow that trains and deploys a model on Azure Machine Learning gets triggered. 
+
+Check the actions tab to view if your actions have successfully run.
 
 ### 5. Modify the code
 
-Now you can start modifying the code in the <a href="/code">`code` folder</a>, so that your model and not the provided sample model gets trained on Azure. Where required, modify the environment yaml so that the training and deployment environments will have the correct packages installed in the conda environment for your training and deployment.
-Upon pushing the changes, actions will kick off your training and deployment run. Check the actions tab to view if your actions have successfully run.
+Now you can start modifying the code in the <a href="/code">`code` folder</a>, so that your model and not the provided sample model gets trained on Azure. 
+
+Where required, modify the environment yaml so that the training and deployment environments will have the correct packages installed in the conda environment for your training and deployment.
+
+Upon pushing the changes, actions will kick off your training and deployment run. 
+
+Check the actions tab to view if your actions have successfully run.
 
-Comment lines 39 to 55 in your <a href="/.github/workflows/train_deploy.yml">`"/.github/workflows/train_deploy.yml"` file</a> if you only want to train the model. Uncomment line 7 to 8, if you only want to kick off the workflow when pushing changes to the `"/code/"` file.
+OPTIONAL: Comment lines 39 to 55 in your <a href="/.github/workflows/train_deploy.yml">`"/.github/workflows/train_deploy.yml"` file</a> if you only want to train the model. Uncomment line 7 to 8, if you only want to kick off the workflow when pushing changes to the `"/code/"` file.
 
 ### 6. Viewing your AML resources and runs
 

```

Comment:
Thanks for the details. That helps users a lot. Good job!
This is great. Thanks!